stanford/kibana.yml

```diff
@@ -1,18 +1,60 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: provision.yml
+  vars:
+    Cluster: kibana
+    keypair: deployment  # TODO: change to stage
+    instance_type: t2.micro
+    root_ebs_size: 8
+    zone: 'us-west-1a'
+    subnet_cidr: '10.0.110.0/24'
+    security_group_rules_ingress:
+    - proto: tcp
+      from_port: 443
+      to_port: 443
+      cidr_ip: 0.0.0.0/0
+    - proto: tcp
+      from_port: 80
+      to_port: 80
+      cidr_ip: 0.0.0.0/0
+    - proto: tcp
+      from_port: 22
+      to_port: 22
+      cidr_ip: 10.0.0.0/16
+    elb_scheme: 'internet-facing'
+    elb_listeners:
+      - protocol: https # options are http, https, ssl, tcp
+        load_balancer_port: 443
+        instance_port: 80
+        ssl_certificate_id: "{{ ssl_certificate_id }}"
+        instance_protocol: http
+    elb_health_check:
+      ping_protocol: tcp
+      ping_port: 80
```

Comment:
Is this where we will update the health check?
The one to ping the elastic search port is /heartbeat